package.json

```diff
@@ -56,6 +56,7 @@
     "autolinker": "^4.0.0",
     "browserify": "^17.0.0",
     "electron-context-menu": "^3.5.0",
+    "eslint-plugin-vuejs-accessibility": "^1.2.0",
```

Comment:
This eslint plugin should be listed as a development dependency, as it is only needed while developing, not while you are actually using FreeTube. I'm alright with browserify being in the dependencies because the polyfills it provides are included in the web builds, but eslint stuff is really only used during development.